test/strong_migrations_test.rb

```diff
@@ -213,12 +213,18 @@ def version
   end
 end
 
-class Custom < TestMigration
+class AddForeignKey < TestMigration
   def change
     add_foreign_key :users, :other
   end
 end
 
+class Custom < TestMigration
+  def change
+    add_foreign_key :users, :other, validate: false
```

Comment:
I ended up getting away with using a similar example for the "Custom" scenario since built-in checks are run first.

Reply:
Would be good to change the custom example to something totally different, like `add_column` with a specific name. I can do this afterwards though.

Reply:
If that's the case, then I would also want to always mark this method as unsafe for older versions, right?